app/assets/stylesheets/modules/_client.scss

```diff
@@ -320,3 +320,14 @@ $client-match-action-row-height: 22px;
   background-color: $brand-info;
   border-color: $brand-info;
 }
+
+.client-notes
+{
+}
+
+.client-notes__attribution
+{
+  margin: 0.5em 0 0;
+  font-size: 13px;
```

Comment:
Let's use the built-in font sizing ` .small` instead of specifying here.  If that' isn't good enough, let's use ems instead of px.